<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountantScreen</class>
 <widget class="QMainWindow" name="AccountantScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Окно бухгалтера</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>75</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>До:</string>
    </property>
   </widget>
   <widget class="QTableView" name="ChecksTableView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>591</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="NextPageBtn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>460</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Вперёд</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>45</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>От:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Даты:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="FromDateDE">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="ToDateDE">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="PrevPageBtn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>460</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RefreshTableBtn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Обновить</string>
    </property>
   </widget>
   <widget class="QLabel" name="PageNumberLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>460</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>679</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Пользователь</string>
    </property>
    <addaction name="ChangeUserAction"/>
    <addaction name="ExitAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Экспорт</string>
    </property>
    <addaction name="HTML_Action"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="ChangeUserAction">
   <property name="text">
    <string>Сменить пользователя</string>
   </property>
  </action>
  <action name="ExitAction">
   <property name="text">
    <string>Выйти</string>
   </property>
  </action>
  <action name="HTML_Action">
   <property name="text">
    <string>Выгрузить в HTML</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
